src/storage/btree.c

```diff
@@ -30309,7 +30292,7 @@ btree_merge_node_and_advance (THREAD_ENTRY * thread_p, BTID_INT * btid_int, DB_V
 
   if (is_system_op_started)
     {
-      /* Abort system operation, before unfixing the pages to be sure that no other transaction modify the pages. */
+      /* Abort system operation. */
```

Comment:
revert all b-tree changes. this actually looks like an older version of btree.c .